kytos/utils/client.py

```diff
@@ -78,6 +78,35 @@ def get_napp(self, username, name):
                                  username, name, res.status_code, res.reason)
         return json.loads(res.content)
 
+    def reload_napps(self, napps=None):
+        """Reload a specific NApp or all Napps.
+
+        Args:
```

Comment:
Hi @llgoncalves could you update the docstrings?

Reply:
Sure, I had forgotten to update. Thanks!